lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -116,16 +116,15 @@ abstract class SqlParserTestBase : TestBase() {
     private fun unwrapQuery(statement: PartiqlAst.Statement) : SexpElement {
        return when (statement) {
            is PartiqlAst.Statement.Query -> statement.expr.toIonElement()
-           is PartiqlAst.Statement.Dml -> statement.toIonElement()
+           is PartiqlAst.Statement.Dml,
            is PartiqlAst.Statement.Ddl,
            is PartiqlAst.Statement.Exec -> statement.toIonElement()
         }
     }
 
     /**
      * Performs checks similar to that of [serializeAssert]. First checks that parsing the [source] query string to
-     * a [PartiqlAst] and to an IonValue Sexp equals the [expectedSexpAst]. Next checks that converting this IonValue
-     * Sexp to an ExprNode equals the [actualExprNode].
+     * a [PartiqlAst] and to an IonValue Sexp equals the [expectedIonSexp].
```

Comment:
For the comment https://github.com/partiql/partiql-lang-kotlin/pull/462#discussion_r739536808.